src/services/profile/index.jsx

```
@@ -0,0 +1,41 @@
+
+        <div
+          className={`text-gray-600 dark:text-gray-400 break-words whitespace-pre-line`}
+        >
+          {subTitle}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const CountCard = ({ count = 0, title }) => (
```

ESLint

[error] 'count' is missing in props validation

(react/prop-types)